static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+              "example": "^/us/.*$"
+            },
+            "serverUrl": {
+              "type": "string",
+              "description": "Origin/server URL from where the html content needs to be fetched to run optimizations.",
+              "example": "something.com"
+            },
+            "deviceType": {
+              "type": "string",
+              "description": "Type of device the optimizations are intended to be applied.",
+              "example": "Desktop"
+            },
+            "optimizationLevel": {
+              "type": "string",
+              "description": "Optimization category to decide which optimizations needs to be applied.",
+              "example": "custom"
+            },
+            "optimizers": {
+              "type": "object",
+              "properties": {
+                "preloadDNSPrefetchHints": {
+                  "type": "boolean",
+                  "example": false
+                },
+                "inlineOutlineJS": {
+                  "type": "boolean",
+                  "example": false
+                },
+                "inlineOutlineCSS": {
+                  "type": "boolean"
+                },
+                "minifyCSS": {
+                  "type": "boolean"
+                },
+                "minifyJS": {
+                  "type": "boolean"
+                },
+                "minifyHTML": {
+                  "type": "boolean"
+                },
+                "bundleJSCSS": {
+                  "type": "boolean"
+                },
+                "predictivePrefetch": {
+                  "type": "boolean"
+                },
+                "preconnectResources": {
+                  "type": "boolean"
+                },
+                "collapseWhitespace": {
+                  "type": "boolean"
+                },
+                "optimizationsByUserAgentType": {
+                  "type": "boolean",
+                  "example": false
+                },
+                "compressionForMobileDevices": {
+                  "type": "boolean",
+                  "example": false
+                },
+                "moveScriptTagsFromTopToBottom": {
+                  "type": "boolean",
+                  "example": false
+                },
+                "deferJS": {
+                  "type": "boolean"
+                },
+                "delayJSExecution": {
+                  "type": "boolean"
+                }
+              }
+            }
+          }
+        },
+        "UpdatePolicyRequest": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the policy to be created.",
+              "example": "Updated Policy Name"
+            },
+            "pageType": {
+              "type": "string",
+              "description": "Type of page to be optimized.",
```

Comment:
Is this a know set of optimization?  Could you use an enum to tell which options are put here?